core/src/main/java/com/github/jsonldjava/core/JsonLdApi.java

```diff
@@ -1968,48 +1977,61 @@ public boolean isWellFormedListNode() {
             }
 
             // 3.5)
+            final Map<String, List<Node>> nodes = new HashMap<>();
+
             for (final RDFDataset.Quad triple : graph) {
                 final String subject = triple.getSubject().getValue();
                 final String predicate = triple.getPredicate().getValue();
                 final RDFDataset.Node object = triple.getObject();
+                final List<Node> list = nodes.getOrDefault(subject, new ArrayList());
```

Comment:
Can this like be switched from ``getOrDefault(subject, new ArrayList())`` to ``computeIfAbsent(subject, ArrayList::new)`` (or similar)? The call to ``nodes.put(subject, list)`` below is then redundant because unlike ``getOrDefault``, ``computeIfAbsent`` modifies the map if necessary before returning.